packages/components/src/components/hds/app-footer/status-link.ts

```diff
@@ -62,7 +71,8 @@ export default class HdsAppFooterStatusLinkComponent extends Component {
    */
   get statusIcon() {
     if (this.status && !this.args.statusIcon) {
-      return STATUSES[this.status].iconName;
+      const index = Object.keys(STATUSES).indexOf(this.status);
+      return STATUSES[index].iconName;
```

Comment:
We should probably add a null check before trying to access `iconName`
```suggestion
      return STATUSES[index]?.iconName;
```

Reply:
Thank you! I was stumbling a bit on this part.